scripts/build-carthage.sh

```diff
@@ -30,6 +30,8 @@ else
   echo "Building Carthage for [$BUILD_CONTEXT]..."
 fi
 
+  ./scripts/fetchNYPLAxis.sh
```

Comment:
I think you want to do this only `if [ "$1" != "--no-private" ]`, because the non-DRM build uses that flag to build

Reply:
although I see now that the non-DRM build succeeded...

Reply:
as we discussed, this will not work in a CI context so I would put this behind something like:

```
if [ "$BUILD_CONTEXT" != "ci" ] && [ "$1" != "--no-private" ]; then
```

Reply:
done. 
Thanks 🙏